app/controllers/clients_controller.rb

```diff
@@ -124,10 +96,9 @@ def set_include
     end
   end
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_client
     # @client = Client.query_filter_by(:symbol, params[:id]).first
-    @client = Client.find_by_id params[:id]
+    @client = Client.find(params[:id])
```

Comment:
@mfenner For example, here using `.find()` gets tricky, because it's case sensitive. so "ANDS" won't work but "ands" will work. I actually was thinking that this method `.find()` doesn't work for the ElasticSearch:persistent under the *ES.ActiveRecord* paradimg. You are trying to use *ES.Repository*, right?

Reply:
I want to use ES.ActiveRecord. We could downcase `params[:id]`.